packages/lit-ssr/src/lib/lit-element-renderer.ts

```diff
@@ -55,6 +55,11 @@ export class LitElementRenderer extends ElementRenderer {
       }
       yield '</style>';
     }
+    // Call LitElement's `willUpdate` method.
+    // Note, this method is required not to use DOM APIs.
+    const changedProperties = new Map();
```

Comment:
Should use the correct `changedProperties` map from the instance, as they keys identify properties that were changed. It seems likely that initialize-state-from-props patterned code will check `changeProps` for existence of a `prop` edge that might cause state to reset.

Reply:
In thinking about whether `_changedProperties` needs to be in `stableProperties`, earlier I said that it only needs to have a helper in `private-ssr-support.js` which gets compiled along with `LitElement` and exposes a non-mangled API, and so technically the property doesn't need to be stable for `lit-ssr` to access it.  However, we need to decide whether `lit-ssr` itself can rely on only ever loading one copy of the core libs, or whether the one it loads needs to be compatible with all versions. In which case any private symbols accessed in `private-ssr-support` do need to be stable (not all of `lit-html`'s are, currently).

Right now, `importModule` in `lit-ssr` does a pretty hardcore workaround of forcing all `lit-html` and `lit-element` packages to resolve to the one it depends on, but I'm not sure if we're going to be able to rely on that long-term, depending on whether we require the module sandbox for all use cases or not.

cc @justinfagnani for thoughts; this is something we need to get right or else risk non-version-compatibility down the road...

Reply:
Made this a reserved property. It was previously private so it had to change.

Reply:
Per https://github.com/Polymer/lit-html/issues/1554, we'll address the question of how version-safe private-ssr-support needs to be in that issue, so can revert or leave this change, either way.